pyblish_qml/control.py

```diff
@@ -1207,7 +1202,7 @@ def iterator(plugins, context):
 
         message = test(**state)
         if message:
-            return StopIteration("Stopped due to %s" % message)
+            raise RuntimeError("Stopped due to %s" % message)
```

Comment:
We probably can't raise here. The iterator used above expects to never fail and always yield a value. Now that it throws, we need a try/except to handle the case where it doesn't yield and instead throws an exception.

I think we should be able to instead..

```py
yield StopIteration("...")
return
```

That should replicate what it was already doing before this yielding thing became an issue.

Reply:
Updated to use suggested syntax.